frontend/src/components/profiles/UserProfile.js

```diff
@@ -32,35 +30,35 @@ function UserProfile() {
                     <div className="active_user_items">
                         <p>Active</p>
                         <div className="list_active_user_items">
-                            <pre className="tab">Item_name               Issue_date:             Return_date:              Due_date: </pre>
+                            <pre className="tab">Item_name Issue_date: Return_date: Due_date: </pre>
```

Comment:
why <pre> 
how you will add name, date and due date
don't use pre tag 
use dummy to fill the fields 
make separate tags for name, date and due date